docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -47,7 +47,7 @@ code: |
     if ( wants_dockets_first ):
       # User has not seen docket sheet instructions yet
       if ( not request_docket_sheets ):
-        certified_copies_page_text_question
+        request_docket_sheets
```

Comment:
This won't cause an error, but doesn't quite make sense either. The previous line already took the user to this question, so this line won't do anything. The previous line could just be `request_docket_sheets` without the if statement. Maybe you meant `if ( not_defined( request_docket_sheets ):`? (Or whatever the name of that function is.)

Reply:
Though I think that's not what we want, right?

Reply:
To clarify, I'm trying to figure out why that `if` statement is there. Right now it serves no purpose, so what is it hoping to do?

Reply:
Just to let you know, I'm not going to request a change for this. It isn't broken. We'll just need to get on the same page about what the purpose of the logic is, so I was thinking that conversation might as well start now. It can be merged as soon as the flow is corrected.

Reply:
Gotcha! The if statement shouldn't be there -- it's a holdover from the previous way this code block was written. It'll probably get corrected in the latest push anyways because I have to revamp this flow based on the Tuesday meeting.